sources/packages/web/src/components/students/modals/ConfirmEditApplication.vue

```diff
@@ -1,12 +1,11 @@
 <template>
   <modal-dialog-base title="Edit application" :showDialog="showDialog">
     <template #content>
-      Editing your application will result in a new assessment, which could
-      delay your application. <strong>Are you sure you want to proceed?</strong>
+      {{ warningText }}
     </template>
     <template #footer>
       <footer-buttons
-        primaryLabel="Edit application now"
+        :primaryLabel="confirmationText"
```

Comment:
Not a blocker, variable names could be `confirmationLabel` and `confimationText`.

Reply:
"primaryLabel" is a standard part of the footer-buttons

Reply:
I did not mean the primaryLabel which is is attribute name. I meant the variable confirmationText. 
![image](https://github.com/user-attachments/assets/1d95d3ab-253d-466e-802c-53e2562441e4)
It is not a blocker and feel free to nor do it.